### PR TITLE
FEAT: added wishlist page (without functionality)

### PR DESCRIPTION
**Pull request by zoyatanweer**
- added wishlist page (without functionality)
- added route for wishlist page in "App.js" file

Files to check :

- [ ] components/wishlist/Wishlist.jsx
- [ ] components/wishlist/WishlistData.js

### CODE REVIEW

**Review by rutu14 — APPROVED** [review]
The code is good to merge.

File: src/App.js [hunk 1]
```diff
@@ -13,6 +14,7 @@ function App() {
         {/* <Route path="/home" element={<Homepage />} /> */}
```
Comment (zoef-yt) [inline src/App.js:14]: Remove commented code.

File: src/components/wishlist/Wishlist.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
+/* wishlist */
+.wishlist-products {
+  display: flex;
+  flex-direction: row;
+  gap: 7rem;
+  margin-left: 4rem;
+}
```
Comment (zoef-yt) [inline src/components/wishlist/Wishlist.css:7]: Inconsistent spacing in the CSS file

**Review by zoef-yt — APPROVED** [review]
Small pr. The code looks good. Good work Zoya. Good to merge.

File: src/components/wishlist/Wishlist.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+/* badge-on wishlist */
+.btn-icon-wishlist,
+.btn-icon-cart {
+  background-color: var(--primary-color);
+  border: solid var(--primary-color);
+  color: var(--background-color);
+  font-size: 1.5rem;
+  /* top:7px; */
```
Comment (zoef-yt) [inline src/components/wishlist/Wishlist.css]: remove commented code

**Comment by zoyatanweer** [comment]
Thankyou for the review guys.

**Pull request merged**